### DISCUSSION TITLE
What is MkDocs current representation within social media?

### DISCUSSION THREAD

**Discussion by Andre601** [mkdocs/mkdocs#3678] [Q&A]
This is a question that has been on my mind for a little while now...

I was wondering how and to what extend MkDocs is represented within social media. Looking around, I can see there is a [Twitter Account]() but it seems somewhat abandoned as the last activity on it was from 2017 by the looks of it.
There also was (keyword) a Mastodon account on chirp.social that was used for automatic reblogging, but that instance has been discontinued... And a real account by MkDocs was this not either...

So the question again is: How is MkDocs represented within the social media area?
I feel like a decent social media representation (i.e. with regular posts about updates and similar) should be considered here, as it can help in maintaining a good community...
Tho, obviously, it's also work. 😅

I'm generally just curious on what platforms I could find MkDocs.

**Comment by pawamoy** [comment]
That's work indeed :sweat_smile: 

But I agree it would be nice if the Twitter account could be resuscitated, or better yet, migrated to Mastodon. Also you're right, chirp.social has been discontinued :confused: I moved it to Guppe though, you can follow it at @mkdocs@a.gup.pe (example: https://fosstodon.org/@mkdocs@a.gup.pe). I however never find time to post to this group. But anyone can if they want :slightly_smiling_face:

**Reply by Andre601** [reply]
I would like to help manage the social media in the future...
Moving to mastodon probs would be better, especially if multiple people should manage it, given that Twitter's Tweetdeck now is paid to be used.
